.github/workflows/debian_package_kanidm.yml

```diff
@@ -19,18 +19,20 @@ jobs:
       matrix:
         image:
           - ubuntu:22.04
-          - ubuntu:20.04
-          - ubuntu:18.04
-          - debian:11
-          - debian:10
+          # - ubuntu:20.04
```

Comment:
Should we comment why these are commented? Or just remove them?

Reply:
oh, yeah, I'll just dump them 😄